### PR TITLE
Bug 2096908: [Dark Theme]: fix several issues in the dark mode.

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes:** https://issues.redhat.com/browse/OCPBUGSM-45453

**Screenshots:**

### **Before:**
issue11__PL_builder

### **After:**
image

### **Before:**
issue41_HelmReleaseRollback

### **After:**
image

### **Before:**
image

### **After:**
image

### CODE REVIEW

File: frontend/packages/helm-plugin/src/components/forms/rollback/HelmReleaseRollbackForm.tsx [hunk 1]
```diff
@@ -39,8 +39,9 @@ const HelmReleaseRollbackForm: React.FC<Props> = ({
 
   const formHelpText = (
     <Trans t={t} ns="helm-plugin">
-      Select the version to rollback <strong style={{ color: '#000' }}>{{ releaseName }}</strong>{' '}
-      to, from the table below:
+      Select the version to rollback{' '}
```
Comment (jerolimov) [inline frontend/packages/helm-plugin/src/components/forms/rollback/HelmReleaseRollbackForm.tsx:42]: Nit: When using a single string at the beginning we don't need to update the translation.

```suggestion
      {'Select the version to rollback '}
```

**Review by jerolimov — APPROVED** [review]
In general, looks good to me. I added one note how we can avoid that the translation changes from `< 1 >` to `< 2 >`. Wdyt?

/approve

File: frontend/packages/helm-plugin/src/components/forms/rollback/HelmReleaseRollbackForm.tsx (hunk 1 above)
Comment (vikram-raj) [reply frontend/packages/helm-plugin/src/components/forms/rollback/HelmReleaseRollbackForm.tsx]: Updated it as per your suggetion.

**Pull request merged**